elementary/monitor/alerts/alert.py

```diff
@@ -24,16 +24,18 @@ def __init__(
             status: str = None,
             subscribers: Optional[List[str]] = None,
             slack_channel: Optional[str] = None,
+            slack_timezone: Optional[str] = None,
             **kwargs
     ):
         self.id = id
         self.elementary_database_and_schema = elementary_database_and_schema
         self.detected_at_utc = None
         self.detected_at = None
+        self.slack_timezone = slack_timezone
```

Comment:
The timezone is part of the alert content and agnostics to the platform it is shared on.
So I think that a better name is just timezone or alert_timezone